packages/devtools_app/lib/legacy_main.dart

```diff
@@ -0,0 +1,148 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
For consistency with the rest of the code base how about call this 
html_main.dart
or 
html_legacy_main.dart if you want.

Reply:
Done.